jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml

```diff
@@ -19,9 +19,10 @@
         - email-ext:
             recipients: '{emails}'
         - gcs-uploader
+    # Need a minimum of 8 kube-system pods ready before declaring cluster ready
```

Comment:
sorry to continue to nit over comments, but this doesn't really explain where the value of 8 comes from.